tidy3d/components/simulation.py

```diff
@@ -2805,6 +2805,34 @@ def warn(istruct, side):
 
         return val
 
+    def _scientific_notation(fmin_med: float, fmax_med: float) -> Tuple[str, str]:
```

Comment:
I guess this shouldn't be a Simulation level method. Maybe Tidy3dBaseModel? Let's also change the variable names to not specifically refer to frequency, more like `min_val, max_val`? And then regarding @tomflexcompute's point, add an extra argument like `min_digits=4`?  Looking at the code it seems that this number would refer not to the total number of significant digits, but to the digits after the decimal, e.g. `1.1234e12 to 250.1234e12` right? I think that's fine.